packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,84 @@
+import Tippy from "@tippyjs/react";
+import clsx from "clsx";
+import * as React from "react";
+import styles from "./Tooltip.module.css";
+
+// Full list of Tippy props: https://atomiks.github.io/tippyjs/v6/all-props/
+type TooltipProps = {
+  /**
+   * The trigger element the tooltip appears next to.
+   */
+  children?: React.ReactElement;
+  /**
+   * The trigger element the tooltip appears next to.
+   *
+   * Use this instead of `children` if the trigger element is being
+   * stored in state or a ref. Most cases will use `children` and not
+   * `reference`.
```

Comment:
Nice prop doc!

Did the idea/reference/wording for storing an element in `state` come from us or from Tippy?

For my own knowledge, I'm curious as to the intended use case. Ref makes sense, but storing an element in state isn't usually necessary (and according to my own mental pattern matching would still work with `children` anyway).

Reply:
Think @dierat deserves all the praise for the work, I'm just migrating to EDS haha. Any thought's on Andrew's question?

Reply:
I don't remember exactly where "state" came from here; I'm fine with dropping it. This is we're currently using `reference` in the platform (and why we added this documentation): https://github.com/FB-PLP/traject/blob/7ab73ba57ce427d8d9b82ec587d89e6bd8121e8d/app/assets/javascripts/v2/teachers/schoolData/components/MetricCard/HorizontalBar.jsx#L171

Reply:
Makes sense. I say lets drop reference to `state`.